.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Apache Celix Dev Container",
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
+    "postCreateCommand": "sh .devcontainer/setup.sh"
```

Comment:
```suggestion
     "runArgs" : [
        "--userns=keep-id"
    ],
    "workspaceMount": "type=bind,src=${localWorkspaceFolder},target=/workspace,z",
    "workspaceFolder": "/workspace",
    "postCreateCommand": "sh .devcontainer/setup.sh"
```

Reply:
Tested on fedora 39/40. 
With the previous devcontainer.json, this resulted in permission errors due to SELinux. 
To mitigate this, setting the workspaceMount with the flag  ```z```.
Will try to test this on ubuntu/debian as well.

Reply:
Thanks for testing this and providing a commit suggestion 😄 .